Makefile

```diff
@@ -11,6 +14,10 @@ tests-run-integration: ## run integration tests.
 	@echo ">>> Running integration tests."
 	go test -v -p 1 -tags="integration" ./tests/integration/...
 
+PHONY: tests-mlflow
```

Comment:
can you please rename `tests-mlflow` -> `paython-tests-run-integration` and please rename existing targets: `tests-run-unit` and `tests-run-integration` to be `go-tests-run-unit` and `go-tests-run-integration` to be consistent.

also please add a comment like `tests-mlflow: build` -> `tests-mlflow: build ## run python integration tests.`. In that case when you call `make help` you will see useful help message for each target.

Reply:
I'd suggest "test" should be the first the token of these targets -- so like "test-go-unit", "test-python-integration"? Also as a verb, "test", like "build" or "clean," instead of "tests".